src/app/manager-dashboard/reports/reports.service.ts

```diff
@@ -41,7 +41,7 @@ export class ReportsService {
       this.couchService.findAll('child_users').pipe(map((users: any) => users.filter(user => user.planetCode === planetCode)));
     return obs.pipe(map((users: any) => {
       return ({
-        count: users.length,
+        count: users.filter(user => user.name !== 'satellite').length,
```

Comment:
Rather than filter just for the count property of the return statement, you should add a new line between lines 42 and 43 to filter the users variable `users = users.filter(...)`.  That way both `count` and `byGender` properties to not include the satellite user.

Reply:
Ok, got it. Thanks @paulbert!